bin.js

```diff
@@ -31,3 +34,12 @@ function getConfig() {
   }
 }
 
+function getRules(set) {
+  var configFileName = set ? ('eslint-config-' + set) : null
```

Comment:
I wonder if we could find how ESLint resolves these `extends` because you can actually have files extend others [like so](https://github.com/kentcdodds/eslint-config-kentcdodds/blob/4b6b00a36a2ba10edbf4a6454625a794949086f7/react.js#L2). So we can either recreate the behavior or see if we can use ESLint's implementation somehow.